webplugin/template/chat.html

```diff
@@ -48,8 +49,9 @@
                 if (skipRouting){
                     defaultSettings["skipRouting"] = !!skipRouting;
                 }
-                
+
                 var kommunicateSettings = {
+                    "inputFocus" : isInputFocussed == "true" ? true : false,
```

Comment:
isInputFocussed == "true" itself returns true or false
so '? true: false' is not required

Reply:
Sir, this is corrected